app/controllers/application_controller.rb

```diff
@@ -5,11 +5,7 @@ class ApplicationController < ActionController::Base
   # user, we have no access to the devise object. So, we need to override
   # current_user to return the current account. This is needed both
   # in ApplicationController and in Admin::ApplicationController
-  before_action :get_alert, :set_footer
-
-  def current_user
-    current_account
-  end
+  before_action :get_alert, :set_footer, :set_paper_trail_whodunnit
```

Comment:
Aren't get alert and set footer just used for display? Do they need whodunnits?

Reply:
@bibliotechy According to the documentation, this is where it's supposed to go